temporal-kotlin/build.gradle

```diff
@@ -1,17 +1,17 @@
 plugins {
-//    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
 //    id 'org.jetbrains.kotlin.jvm' version '1.4.21'
-    id 'org.jetbrains.kotlin.jvm' version '1.5.21'
+//    id 'org.jetbrains.kotlin.jvm' version '1.5.21'
 
     id 'org.jlleitschuh.gradle.ktlint' version '10.1.0'
 }
 
 description = '''Temporal Workflow Java SDK Kotlin'''
 
 ext {
-//    kotlinVersion = '1.3.72'
+    kotlinVersion = '1.3.72'
 //    kotlinVersion = '1.4.21'
-    kotlinVersion = '1.5.21'
+//    kotlinVersion = '1.5.21'
```

Comment:
Are these minimums? Why the different commented out ones? Just testing compat?

Reply:
It's the last bugfix releases for all minor versions that are still popular and keeps updating.
The original code I wrote worker on 1.4, but didn't work on 1.5 for example, so yeah, I leave it for easier testing